dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -21,5 +21,13 @@ role.administrator.desc=A collection of users and/or groups assigned the server
 role.user=User name
 role.user.desc=User assigned a role.
 
+role.userAccessId=User access ID
+role.userAccessId.desc=A user access ID in the general form user:realmName/userUniqueId. 
```

Comment:
What's meant by "general" form? What about: A user access ID in the format user:realmName/userUniqueId.

Reply:
The metatype property already deliver in the PR# https://github.com/OpenLiberty/open-liberty/pull/3627